synfig-studio/src/gui/widgets/widget_timeslider.cpp

```diff
@@ -338,9 +365,20 @@ Widget_Timeslider::on_button_press_event(GdkEventButton *event) //for clicking
 
 	if (event->button == 1) {
 		Time time = time_plot_data->get_t_from_pixel_coord(event->x);
-		time_plot_data->time_model->set_time(time);
+		time_plot_data->time_model->set_time(time);//setting the time which is in time model
 	}
 
+	double current= time_plot_data->get_double_pixel_t_coord(time_plot_data->get_t_from_pixel_coord(event->x));
```

Comment:
surround assignment operator `=` with single space

Reply:
excuse all these not spaced out operators I wrote them before knowing about this style xD